projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -144,52 +144,8 @@ export default function VideoPressEdit( {
 	const token = useMediaToken( { guid, isPrivate, tokenType: 'playback' } );
 
 	// Store if the chapters file is auto-generated in a local state.
-	const [ isAutoGeneratedChapter, setIsAutoGeneratedChapter ] = useState( true );
-
-	// Detect if the chapter file is auto-generated.
-	const chapter = tracks?.filter( track => track.kind === 'chapters' )?.[ 0 ];
-
-	useEffect( () => {
-		if ( ! chapter || ! chapter.src ) {
-			return;
-		}
-
-		// Wait for the token when the video is private.
-		if ( isPrivate && ! token ) {
-			return;
-		}
-
-		let queryString = '';
-		if ( token ) {
-			queryString = '?' + new URLSearchParams( { metadata_token: token } ).toString();
-		}
-
-		const chapterUrl =
-			'https://videos.files.wordpress.com/' + guid + '/' + chapter.src + queryString;
-
-		try {
-			fetch( chapterUrl )
-				.then( function ( response ) {
-					response.text().then( function ( text ) {
-						if ( ! text ) {
-							return;
-						}
-
-						const isAutoGeneratedChaptersFile = /videopress-chapters-auto-generated/.test( text );
-						if ( isAutoGeneratedChaptersFile ) {
-							debug( 'Chapter %o auto-generated detected', chapter.src );
-						}
-						setIsAutoGeneratedChapter( isAutoGeneratedChaptersFile );
-					} );
-				} )
-				.catch( function ( error ) {
-					// eslint-disable-next-line no-console
-					console.error( error );
-				} );
-		} catch ( error ) {
-			throw new Error( error?.message ?? error );
-		}
-	}, [ chapter, guid, token, isPrivate ] );
+	const chapter = tracks?.find( ( track: TrackProps ) => track.kind === 'chapters' )?.[ 0 ];
```

Comment:
I tested the auto-generated chapters and noticed that they weren't working. Seems the cause is in this function as we changed the function to look for chapters from `filter` to `find`. However, we are still trying to get the first item which results in `undefined` always.

Hence, we either keep the usage of `filter`:
```suggestion
	const chapter = tracks?.filter( ( track: TrackProps ) => track.kind === 'chapters' )?.[ 0 ];
```

Or use `find`:
```suggestion
	const chapter = tracks?.find( ( track: TrackProps ) => track.kind === 'chapters' );
```